File: issta2018artifacts/minimized-bugs/buggy-key/OptionsToHtml.java
// simplified version of bug in plume-lib fixed by b5df093053146bcf32f485eea974e70cc854b407

public class OptionsToHtml {

    /*@ public normal_behavior
@ requires true;
@ ensures true;
@*/
    public void optionsToHtml(boolean includeClassDoc, Object... root_classes) {
	if (includeClassDoc) {
	    //:: error: (array.access.unsafe.high.constant)
	    javadocToHtml(root_classes[0]);
	}
    }

    /*@ public normal_behavior
@ requires true;
@ ensures true;
@*/
    public void javadocToHtml(Object o) { }
}

File: issta2018artifacts/minimized-bugs/buggy-clousot/PlumeLib/OptionsToHtml.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

// simplified version of bug in plume-lib fixed by b5df093053146bcf32f485eea974e70cc854b407

public class OptionsToHtml
{
    public void optionsToHtml(bool includeClassDoc, params object[] root_classes)
    {
        if (includeClassDoc)
        {
            //:: error: (array.access.unsafe.high.constant)
            javadocToHtml(root_classes[0]);
        }
    }

    public void javadocToHtml(object o) { }
}
